packages/ckeditor5-utils/src/locale.ts

```diff
@@ -201,6 +201,6 @@ function interpolateString( string: string, values: ReadonlyArray<any> ): string
 export type Translations = {
 	[ language: string ]: {
 		dictionary: { [ messageId: string ]: string | ReadonlyArray<string> };
-		getPluralForm?: ( n: number ) => number;
+		getPluralForm: ( ( n: number ) => number | boolean ) | null;
```

Comment:
Where is it used that way?

Reply:
The English translation generated by the new CLI contains `getPluralForm: null`. The same translation generated by the old tool skips this field entirely, so I reverted the `?` symbol.